src/main/java/de/tum/in/www1/artemis/security/jwt/TokenProvider.java

```diff
@@ -85,6 +68,16 @@ public void init() {
         this.tokenValidityInMillisecondsForRememberMe = 1000 * jHipsterProperties.getSecurity().getAuthentication().getJwt().getTokenValidityInSecondsForRememberMe();
     }
 
+    /**
+     * Gets the validity for the generated tokens.
+     *
+     * @param rememberMe     Determines Token lifetime (30 minutes vs 30 days)
```

Comment:
To avoid confusion I suggest to remove the default lifetimes from the comment, as they can be overwritten within the config files, e.g. on the test servers we use 1 day instead of 30 minutes.
```suggestion
     * @param rememberMe     Determines Token lifetime
```

Reply:
Agreed, committed!